jet/utils.py

```diff
@@ -449,8 +450,10 @@ def context_to_dict(context):
     return context
 
 
-def user_is_authenticated(user):
-    if not hasattr(user.is_authenticated, '__call__'):
+if VERSION[0] >= 1 and VERSION[1] >= 10:
```

Comment:
Wait, shouldn't this be `if VERSION[0] > 1 or (VERSION[0] == 1 and VERSION[1] >= 10)`

You want this to work for version 2.1 right?